frontend/device/android/device.lua

```diff
@@ -131,9 +131,17 @@ local Device = Generic:new{
 function Device:init()
     self.screen = require("ffi/framebuffer_android"):new{device = self, debug = logger.dbg}
     self.powerd = require("device/android/powerd"):new{device = self}
+
+    local event_map = require("device/android/event_map")
```

Comment:
did you mean `android_eventmap` here?.

Reply:
i dont know what you mean, the variable is named `event_map` and the file that is imported is `device/android/event_map`, like before the PR

Reply:
no, i mean your variable is named `event_map` but you're asigning new actions to `android_eventmap`, which is `nil`. You need to use the same variable name everywhere.

Reply:
oh, thats what you meant, i think i forgot to change it

Reply:
fixed with 42135c8b3a460aa4be80877ab0b3b61044b61a9c